components/operator/hack/ci/Makefile

```diff
@@ -74,18 +84,20 @@ remove-serverless: ## Remove Serverless CR
 
 ### Internal Dependencies
 
+.PHONY: create-k3d
+create-k3d:
+	@make -C ${PROJECT_COMMON} kyma create-k3d
+
 .PHONY: install-latest-serverless
 install-latest-serverless:
-	@make -C ${PROJECT_COMMON} kyma create-k3d
+	kubectl create namespace kyma-system || true
 	kubectl apply -f https://github.com/kyma-project/serverless-manager/releases/latest/download/serverless-operator.yaml
 	kubectl apply -f https://github.com/kyma-project/serverless-manager/releases/latest/download/default-serverless-cr-k3d.yaml -n kyma-system
 	@make -C ${PROJECT_COMMON} verify-serverless
 
-.PHONY: reinstall-serverless
-reinstall-serverless:
-	@make -C ${PROJECT_COMMON} run-without-lm-on-k3d-cluster
-	
-	# wait some time to make sure lm starts the reconciliation first
+.PHONY: check-serverless-availability
+check-serverless-availability:
```

Comment:
Do we need to have 2 way to check availability of serverless: 
- this target
- verify-serverless from `common` which is called by `run-without-lm-on-cluster`

Reply:
these targets are not the same. The `verify-serverless` checks if Serverless CR is in a ready state and the `check-serverless-availability` checks if the most important deployments are ready for action. The second target is here to avoid situations when (for example) the operator is down after the upgrade and because it's down, the Serverless CR is not touched by a new operator (it's still in a ready state because of the first operator reconciliation) and the serverless component is not upgraded